app/assets/javascripts/tufts/ensure_upload_fade_in.js

```diff
@@ -1,3 +1,7 @@
 $('#fileupload').bind('fileuploadstop', function (e, data) {
-  $('.template-download').addClass('in')
+  $('.template-download').addClass('in');
+  $('.template-upload').addClass('in')
 })
+
```

Comment:
I understand attaching the class in the callback for the upload, but why outside of that scope, below?

Reply:
I think this may be a better scoped way of achieving what's needed --  seems to work locally, running the tests now too:
```Blacklight.onLoad(function() {
  $('#fileupload').bind('fileuploadadd', function (e, data) {
    $('.template-upload').addClass('in');
  });
});
```